wgpu-core/src/logging/subscriber/chrome.rs

```diff
@@ -0,0 +1,67 @@
+use smallvec::SmallVec;
+use std::fmt;
+use tracing::field::{Field, Visit};
+use tracing::{Event, Level, Subscriber};
+use tracing_subscriber::{layer::Context, registry::LookupSpan, Layer};
+
+#[derive(Debug, Default)]
+struct FmtEventVisitor {
+    message: String,
+}
+
+impl Visit for FmtEventVisitor {
+    fn record_debug(&mut self, field: &Field, value: &dyn fmt::Debug) {
+        match field.name() {
+            "message" => self.message = format!("{:?}", value),
+            _ => {}
+        }
+    }
+}
+
+pub struct FmtLayer;
+
+impl<S> Layer<S> for FmtLayer
+where
+    S: Subscriber + for<'span> LookupSpan<'span>,
+{
+    fn on_event(&self, event: &Event<'_>, ctx: Context<'_, S>) {
+        let time = chrono::Local::now().format("%Y-%m-%d %H-%M-%S%.6f");
```

Comment:
I wonder if we should make this optional, even when subscriber feature is ON. We can do all the things without printing the exact date. We can count time using `std::time::Duration`.

Reply:
Eh, I'll just nuke it entirely, I doubt that absolute times matters to graphics applications as much as it does to say web server applications where multiple logs have to be synced.